src/ethereum/berlin/vm/precompiled_contracts/modexp.py

```diff
@@ -0,0 +1,100 @@
+"""
+Ethereum Virtual Machine (EVM) MODEXP PRECOMPILED CONTRACT
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. contents:: Table of Contents
+    :backlinks: none
+    :local:
+
+Introduction
+------------
+
+Implementation of the `MODEXP` precompiled contract.
+"""
+from ethereum.base_types import U256, Bytes, Uint
+from ethereum.utils.byte import right_pad_zero_bytes
+
+from ...vm import Evm
+from ...vm.gas import subtract_gas
+from ..exceptions import OutOfGasError
+
+GQUADDIVISOR = 20
+
+
+def modexp(evm: Evm) -> None:
+    """
+    Calculates `(base**exp) % modulus` for arbitary sized `base`, `exp` and.
+    `modulus`. The return value is the same length as the modulus.
+    """
+    import pytest
+
+    pytest.xfail()
```

Comment:
Is this left-over from some developmental testing?

Reply:
Nope, Peter's just waiting on me ^^;;